portal-web/docroot/html/common/themes/password_expiring_soon.jspf

```diff
@@ -14,13 +14,13 @@
  */
 --%>
 
-<c:if test="<%= themeDisplay.isSignedIn() && UserLocalServiceUtil.isPasswordExpiringSoon(user) %>">
-	<aui:script use="liferay-notice">
-		new Liferay.Notice(
+<c:if test='<%= SessionMessages.contains(request, "passwordExpiringSoon") %>'>
```

Comment:
Sounds good @holatuwol!  I will update and send it along.

Please let me know if you have any questions. Thanks guys!